.github/workflows/test.yml

```diff
@@ -49,27 +58,49 @@ jobs:
           dockerfile: dotnet-aspnet/Dockerfile.${{ matrix.ubuntu-release }}
           ignore: DL3008,DL3015,SC3028
 
+      # Build images for multiple archs
       - name: Build the .NET runtime deps container image
         run: |
-          docker build \
-            -t ${{ env.runtime-deps-image-name }} \
+          set -x
+          archs=`cat rockcraft.*.dotnet-deps-6.0-${{ matrix.ubuntu-release }}.yaml | shyaml get-value platforms | shyaml keys`
+          buildx_platforms="linux/$(echo ${archs} | sed 's/ /,linux\//g')"
+
+          docker buildx build \
+            --platform=${buildx_platforms} \
+            --output type=oci,dest=dotnet-deps.tar \
```

Comment:
I'm not understanding the need to output to a tarball. Why not just build directly to the image in the local daemon?

Reply:
you cannot `--load` multi-arch images into the Docker daemon (not supported yet):

```
error: docker exporter does not currently support exporting manifest lists
```